hw/tests/config_streamer.py

```diff
@@ -8,15 +9,15 @@
 
 # Add the amplipi directory to PATH
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../..')))
-from amplipi.eeprom import BoardInfo, BoardType, EEPROM, UnitType
 
-BOARD_REV_RE = re.compile(r"(\d+)([A-z])") # accept lower or uppser case change designator
-                                           # we will uppercase later
+BOARD_REV_RE = re.compile(r"(\d+)([A-z])")  # accept lower or uppser case change designator
```

Comment:
```suggestion
BOARD_REV_RE = re.compile(r"(\d+)([A-z])")  # accept lower or upper case change designator
```
While we're changing this anyway...